src/components/Resources/ResourcePage.js

```diff
@@ -15,37 +16,27 @@ const useStyles = makeStyles({
 });
 
 function ResourcePage({ matchProps }) {
-  const [resource, setResource] = useState({});
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    axios
-      .get('/api/v1/resources/' + matchProps.match.params.guid)
-      .then(function(response) {
-        // handle success
-        setResource(response.data);
-        setLoading(false);
-      })
-      .catch(function(error) {
-        // handle error
-        console.log(error);
-      });
-  }, [matchProps.match.params.guid]);
-
+  const resourceId = matchProps.match.params.guid;
+  // TODO: Handle Error cases
+  const { isLoading, data } = useQuery(['resource', resourceId], getResource);
```

Comment:
I think we can do `const {isLoading, data, error}` here and later something like:
```
if (error) {
    return <p>{error.detail}</p>
}
```

This is because the response from the API looks like this when there is an error: 
![image](https://user-images.githubusercontent.com/4512699/90328765-d3fbde00-df53-11ea-98d5-c7b5248a13b9.png)

Reply:
@lpatmo - when the API doesn't resolve in DRF, does it return us an error object or does it throw an error? I'm curious how our `getResource()` function should handle error responses? Right now we're just returning a data object:
https://github.com/codebuddies/frontend/blob/700010ad534dc2522b593d091e5d4b515ba41141/src/utils/queries.js#L6-L7

Reply:
Sorry, I should have tried this out before I made the suggestion! DRF throws a 404 error. 

We could add try/catch to that function in queries.js, but if we do that the error that is returned is considered "data" and not "error". 

Wondering how you feel about doing this instead: 

```
  const { isLoading, data, error } = useQuery(
    ['resource', resourceId],
    getResource
  );
  const classes = useStyles();

  if (error) {
    console.error(error);
    return <p>Sorry, there was an error!</p>;
  }
  if (isLoading) {
    return <p>Loading...</p>;
  }
```

Reply:
Gotcha, this then tells me that `getResource` should be returning an error object from Axios then but I think that can be moved to another issue to handle errors a lot more appropriately.